modules/combined/doc/content/newsletter/2018_10_04.md

```diff
@@ -156,6 +156,26 @@ influence matrix conditioning and solution accuracy, and generally
 should be avoided if possible.
 
 
+## Add FEProblem-derived objects via Actions
+
+Custom Problems are advanced MOOSE objects which are useful in several
+different scenarios, including coupling to external codes
+(`ExternalProblem`) and solving eigenvalue problems
+(`EigenProblem`). Prior to the addition of this `Action`, using a
+custom Problem required users to declare a `Problem` block in their
+input file with a custom `type =` field, and this process has now been
+streamlined.  A test of this new capability has been added in
+[tests/problems/action_custom_fe_problem/action_custom_fe_problem_test.i].
+This test uses the dummy `CreateSpecialProblemAction` to add an
+`FEProblem`-derived class of type `MooseTestProblem`. The new capability
+is accessed in the test via the following input file syntax:
+
+```
+[TestProblem]
+  name = 'MOOSE Action Test problem'
+[]
+```
```

Comment:
Code developers can follow the same code pattern within `CreateSpecialProblemAction` to enhance their own actions.

Reply:
yeap. This give experienced people a clue how to use this capability.